search/model.hpp

```diff
@@ -47,6 +47,11 @@ class Model
     TYPE_COUNT
   };
 
+  static bool IsLocalityType(Type const type)
+  {
+    return type >= TYPE_VILLAGE && type <= TYPE_COUNTRY;
```

Comment:
nit: add a note about the dangers of reordering in the Type enum

Reply:
Done